django_tenants/models.py

```diff
@@ -154,7 +154,7 @@ def create_schema(self, check_if_exists=False, sync_schema=True,
                              verbosity=verbosity)
             else:
                 # create the schema
-                cursor.execute('CREATE SCHEMA %s', (self.schema_name,))
+                cursor.execute('CREATE SCHEMA %s' % self.schema_name)
```

Comment:
http://initd.org/psycopg/docs/cursor.html#cursor.execute
This is asking for failure. You should leave processing params to psycopg2, as it map it to proper type using internal mapping system.

Reply:
Found the bug: the branch I originally worked this out on supports schema names with capitalization, so there I'm passing `psycopg2.extensions.AsIs(schema_name)` instead of just `schema_name`. I'd reverted to the old notation here so I could keep the functionality separate. Will fix.

Reply:
Nice one :+1:

Reply:
@sguermond Thank you for hard work. I will merge once you send the latest changes and i have tested them.
@kosz85 Thank you for doing some of the code reviewing.

Reply:
@sguermond did you manage to fix the bug?

Reply:
@sguermond did you manage to fix the bug?